components/form/AccountCoding.tsx

```diff
@@ -18,19 +18,19 @@ export default function AccountCoding({
 
   const [accountCoding, setAccountCoding] = useState({
```

Comment:
I see the same logic below to parse `accountCodingInitial` to produce `accountCoding` object;
I think we can make it as a function to reuse in multiple places.